teachopencadd/talktorials/T003_compound_unwanted_substructures/talktorial.ipynb

```diff
@@ -122,23 +122,17 @@
    "cell_type": "code",
```

Comment:
<p>Please remove empty line before mols2grid as it is part of the third party import block (see the respective <a href="https://peps.python.org/pep-0008/#imports" target="_blank">PEP</a>).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>